videostore/videostore.go

```diff
@@ -187,7 +207,13 @@ func NewFramePollingVideoStore(config Config, logger logging.Logger) (VideoStore
 		logger.Warnf("encoder init failed: %s", err.Error())
 		return nil, err
 	}
-
+	if renamer != nil {
+		vs.renameWorker.Add(func(ctx context.Context) {
+			if err := renamer.processSegments(ctx); err != nil {
+				vs.logger.Errorf("failed to process segments: %v", err)
+			}
+		})
```

Comment:
processSegments never returns an error:
```suggestion
		vs.renameWorker.Add(renamer.processSegments)
```